.github/workflows/percy-visual-label.yml

```diff
@@ -0,0 +1,34 @@
+# Triggers Percy job on pull requests that contain "visual" label
+name: Percy visual tests
+
+on:
+  pull_request:
+    types: [synchronize, labeled]
```

Comment:
TBH. it worries me about the `synchronize`, I feel like we can be running more than we should

Reply:
@diogormendes that was the idea to make it more convenient for subsequent commits. Base branch change events is also an important trigger because Percy comares screenshots with PR base branches. What events do you think we should exclude?

Reply:
@diogormendes yes, engineers should be mindful about it. That's why I did not go with checking PR draft status because it is a general purpose thing. But a special label is something engineers are supposed to assign intentionally and therefore be aware of it. This is my assumption but I might be wrong